test/ae_mdw_web/controllers/tx_controller_test.exs

```diff
@@ -1122,21 +1122,15 @@ defmodule AeMdwWeb.TxControllerTest do
           |> Store.put(Model.PlainName, Model.plain_name(index: name_hash2, value: plain_name2))
           |> Store.put(
             Model.ActiveName,
-            Model.name(
```

Comment:
It seems the controller test was incomplete. Can we have the `PreviousName` records on the fixture and check that it appears on the response?

Reply:
I've added previous names on the auctions test case which is where previous names appear